readme.md

```diff
@@ -164,6 +164,22 @@ function Controls() {
 
 ----
 
+## Installation notes for Yarn 2 users
+
+If you are using yarn v2 (yarn berry), you need to install all peer dependencies by yourself.
```

Comment:
These instructions shouldn't be Yarn 2 specific, anyone using this needs to have these dependencies present

Reply:
As you said in the other issue, yarn 2 (pnp) is especially strict. For other environments, one wouldn't need these instructions. They are often installed implicitly. Personally, I never needed them.

The difference between yarn 2 and others like yarn 1 is not about whether you need those dependencies (because you always need them), but about how dependencies are specified in package.json. These instruction would ease the use case with yarn 2, but for yarn 1 users and npm users, they are not necessary.

Reply:
`jotai` has the same peer dependencies no matter what package manager you're using. I don't mind the instruction, my point is just that the same is true when using npm or pnpm so it shouldn't be Yarn 2 specific as it makes it seem like it's more work to use `jotai` with Yarn 2, which isn't the case.

> They are often installed implicitly. Personally, I never needed them.

I don't think users *usually* install `scheduler`, it just happens to work because hoisting brings it up to an accesible location from `react-dom`. You can read more about hoisting here https://yarnpkg.com/advanced/rulebook#packages-should-only-ever-require-what-they-formally-list-in-their-dependencies

Reply:
Hm, so we shouldn't depend on the hoisting behavior. I don't want to make it seem more work with yarn 2.